bower.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "backbone-events-standalone",
-  "main": "backbone-events-standalone.js",
+  "main": "backbone-events-standalone.min.js",
```

Comment:
I always prefer to load the un-minified version of a library as that makes it easier to debug applications. Minification can be done during the build process of whoever uses this library.